### PR TITLE
Documentation Standard Updates

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
This pull request was created automatically 🎉:

- [x] Ensure `.github/` files have been properly updated (templates are updated and auto releaser is included)
- [x] Confirm all appropriate end models are included in the `.quickstart/quickstart.yml` file as `public_models` (only for transform repos)
- Verify the README has made all necessary updates:
    - [x] Removal of emojis
    - [x] Replacing exclamation points with periods in sentences
    - [x] Adjusting all headers to ensure only one H1 heading
    - [x] Confirming a section header is included for Quickstart UI rendering (only for transform repos)
    - [x] Renaming `**model` to `**table**` (only for transform repos)
    - [x] Removal of Fivetran community and office hour resources (if necessary)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,13 +11,13 @@
...
+## What does this dbt package do?
 - Produces modeled tables that leverage Xero data from [Fivetran's connector](https://fivetran.com/docs/applications/xero) in the format described by [this ERD](https://fivetran.com/docs/applications/xero#schemainformation) and builds off the output of our [Xero source package](https://github.com/fivetran/dbt_xero_source).
 
 - Provides analytics-ready models, including a profit and loss report, general ledger, and balance sheet report.
 - Generates a comprehensive data dictionary of your source and modeled Xero data through the [dbt docs site](https://fivetran.github.io/dbt_xero/).
 
-The following table provides a detailed list of all models materialized within this package by default. 
+The following table provides a detailed list of all models materialized within this package by default.
 
 | **Model**                | **Description**                                                                                      |
```
Comment (fivetran-joemarkiewicz) [inline README.md:22]:
```suggestion
<!--section=“xero_transformation_model"-->

The following table provides a detailed list of all tables materialized within this package by default.

| **Table**                | **Description**                                                                                      |
```

File: README.md [hunk 2]
```diff
@@ -26,15 +26,15 @@ The following table provides a detailed list of all models materialized within t
 | [xero__balance_sheet_report](https://github.com/fivetran/dbt_xero/blob/main/models/xero__balance_sheet_report.sql)    | Each record represents the state of the balance sheet for a given account on a given month.                            |
 | [xero__invoice_line_items](https://github.com/fivetran/dbt_xero/blob/main/models/xero__invoice_line_items.sql)      | Each record represents an invoice line item enriched with the account, contact, and invoice information.                   |
 
-# 🎯 How do I use the dbt package?
+## How do I use the dbt package?
```
Comment (fivetran-joemarkiewicz) [inline README.md:29]:
```suggestion
<!--section-end-->

## How do I use the dbt package?
```

File: README.md [hunk 3]
```diff
@@ -116,14 +120,14 @@ vars:
 
 </details>
 
-## (Optional) Step 5: Orchestrate your models with Fivetran Transformations for dbt Core™ 
+### (Optional) Step 5: Orchestrate your models with Fivetran Transformations for dbt Core™
```
Comment (fivetran-jamie) [inline README.md:124]:
```suggestion
<details><summary>Expand for details</summary>
<br>
```

File: README.md [hunk 4]
```diff
@@ -52,13 +56,13 @@ vars:
     xero_database: your_database_name 
 ```
 
-## (Optional) Step 4: Additional configurations
+### (Optional) Step 4: Additional configurations
```
Comment (fivetran-jamie) [inline README.md:60]: <details open><summary>Expand/Collapse configurations</summary>
<br>

Comment (fivetran-jamie) [inline README.md:60]:
```suggestion
<details open><summary>Expand/Collapse configurations</summary>
<br>
```

File: README.md [hunk 5]
```diff
@@ -116,14 +121,15 @@ vars:
 
 </details>
 
-## (Optional) Step 5: Orchestrate your models with Fivetran Transformations for dbt Core™ 
-
+### (Optional) Step 5: Orchestrate your models with Fivetran Transformations for dbt Core™
+<details><summary>Expand for details</summary>
+<br>
```
Comment (fivetran-jamie) [inline README.md:126]:
```suggestion
<br>

```

File: README.md [hunk 6]
```diff
@@ -52,13 +56,14 @@ vars:
     xero_database: your_database_name 
 ```
 
-## (Optional) Step 4: Additional configurations
-
-### Note about currency gains
+### (Optional) Step 4: Additional configurations
+<details open><summary>Expand/Collapse configurations</summary>
+<br>
```
Comment (fivetran-jamie) [inline README.md:61]:
```suggestion
<br>

```

File: README.md [hunk 7]
```diff
@@ -52,13 +56,15 @@ vars:
     xero_database: your_database_name 
 ```
 
-## (Optional) Step 4: Additional configurations
+### (Optional) Step 4: Additional configurations
+<details open><summary>Expand/Collapse configurations</summary>
+<br>
```
Comment (fivetran-jamie) [inline README.md:61]:
```suggestion
```

**Review by fivetran-jamie — APPROVED** [review]

**Pull request merged**